synthea-hiv/uploader/uploader_test.py

```diff
@@ -101,3 +104,49 @@ def test_fetch_location(self):
             '7f65d926-57d6-4402-ae10-a5b3bcbf7986': 'Pharmacy',
             '7fdfa2cb-bc95-405a-88c6-32b7673c0453': 'Laboratory'
         })
+
+  def _create_file(self) -> tempfile.NamedTemporaryFile:
+    # A Bundle with a single Patient is good enough for most of the tests.
+    config = {
+        "resourceType": "Bundle",
+        "type": "transaction",
+        "entry": [{
+            "fullUrl": "urn:uuid:495b7301-3b1b-9283-0334-8b574c3f9424",
+            "resource": {
+                "resourceType": "Patient",
+                "id": "495b7301-3b1b-9283-0334-8b574c3f9424",
+                "identifier": [{
+                    "system": "https://github.com/synthetichealth/synthea",
+                    "value": "495b7301-3b1b-9283-0334-8b574c3f9424"
+                }],
+                "name": [{
+                    "use": "official",
+                    "family": "Hagenes547",
+                    "given": [ "Andreas188" ],
+                    "prefix": [ "Mr." ]
+                }],
+                "telecom": [ {
+                    "system": "phone",
+                    "value": "555-338-1943",
+                    "use": "home"
+                }],
```

Comment:
Should we have this string as a constant defined at the top of the file in case we want to re-use this example

Reply:
Do you mean the whole `config` (note it is not a single string)? I guess we can move it to a constant when the need arises but currently we actually reuse this in different tests by calling this `_create_file` function. That is the intended reuse pattern for now.

Reply:
yes, I meant the whole config as a constant. This works for now though!